src/main/java/mate/academy/exceptions/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.exceptions;
+
+public class DataProcessingException extends RuntimeException{
+    public DataProcessingException(String message, Throwable ex) {
+        super(message);
```

Comment:
The constructor should pass the `Throwable ex` to the superclass constructor to preserve the exception chaining. Modify the constructor to `super(message, ex);` to include the cause in the exception stack trace.